io/disk/lvsetup.py

```diff
@@ -57,6 +57,13 @@ def setUp(self):
         if self.fs_name == 'xfs':
             pkg = 'xfsprogs'
         if self.fs_name == 'btrfs':
+            if distro.detect().name == 'rhel':
+                if (int(distro.detect().version) == 7 and \
+                   int(distro.detect().release) >= 4) or \
+                   (int(distro.detect().version) > 7 or \
+                   int(distro.detect().version) == 0):
```

Comment:
checking the version is irrelevant I think as  just checking distro name > 7 will serve the purpose

Reply:
@PraveenPenguin  yeah this time, i tested this on failing RHEL8  and the condition (int(distro.detect().version) > 7 alone didn't help.

For RHEL8 which has issue.   distro.name is rhel.  but version and release are 0.

Reply:
# uname -a
Linux ltczz401-lp4.aus.stglabs.ibm.com 4.18.0-141.el8.ppc64le #1 SMP Fri Aug 30 15:17:29 UTC 2019 ppc64le ppc64le ppc64le GNU/Linux

# cat /etc/redhat-release 
Red Hat Enterprise Linux release 8.1 Beta (Ootpa)

# python
Python 2.7.16 (default, Jul  2 2019, 15:57:05) 
[GCC 8.3.1 20190507 (Red Hat 8.3.1-4)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from avocado.utils import distro
>>> distro.detect().name
'rhel'
>>> distro.detect().version
0
>>> distro.detect().release
0

@PraveenPenguin @narasimhan-v